### DISCUSSION TITLE
Checkout Order without Customer adress only email / telephone number

### DISCUSSION THREAD

**Discussion by slamer59** [saleor/saleor#11716] [Q&A]
Hello, 
is it possible to create an order with only an email / phone number ? 
It seems mandatory ?

**Comment by peelar** [comment]
Hello @slamer59 👋 

I am afraid there is currently no way to create an order with just an email or a phone number.

What is your use case?

**Comment by slamer59** [comment]
Hello,
 the idea is to capture the payment with just the minimum of information. Then later, update the information.

**Comment by timuric** [comment]
@slamer59 you could create placeholder emails with phone numbers such as `1234567@saleor`
